src/Router.sol

```diff
@@ -471,14 +506,14 @@ contract Router is ReentrancyGuard, IRouter, ScheduledUpgradeable, AccessControl
         emit VerificationFeeBpsUpdated(verificationFeeBps);
     }
 
-    /// @notice Sets the minimum contract upgrade delay
-    /// @param _minimumContractUpgradeDelay The new minimum delay for upgrade operations
-    function setMinimumContractUpgradeDelay(uint256 _minimumContractUpgradeDelay)
+    /// @notice Sets the minimum delay required for scheduling contract upgrades.
+    /// @param _minimumContractUpgradeDelay The new minimum delay in seconds
+    /// @param signature BLS signature from the admin threshold validating the update
+    function setMinimumContractUpgradeDelay(uint256 _minimumContractUpgradeDelay, bytes calldata signature)
         public
         override (IRouter, ScheduledUpgradeable)
-        onlyAdmin
     {
-        super.setMinimumContractUpgradeDelay(_minimumContractUpgradeDelay);
+        super.setMinimumContractUpgradeDelay(_minimumContractUpgradeDelay, signature);
     }
```

Comment:
I somehow thought that function had already been fixed to be onlyCommittee a while back 😮